src/plugins/openni/skelgui/texture_drawer.cpp

```diff
@@ -140,14 +140,14 @@ SkelGuiTextureDrawer::copy_rgb_to_texture(const unsigned char *rgb_buf)
   unsigned char *row = texture_;
   unsigned char *tex = texture_;
   const unsigned char *rgb = rgb_buf;
-  unsigned int bytes = 0;
+  //unsigned int bytes = 0;
```

Comment:
Why not remove it completely?

Reply:
This is part of some debug function that allows to count bytes. But in this case, since the debug output itself is no longer in there, I agree to remove it.